deploy/consts.ts

```diff
@@ -69,24 +45,82 @@ export const L1_ADDRESS_MAP: { [key: number]: { [contractName: string]: string }
     scrollERC20GatewayRouter: "0x13FBE0D0e5552b8c9c4AE9e2435F38f37355998a",
     scrollMessengerRelay: "0x50c7d3e7f7c656493D1D76aaa1a836CedfCBB16A",
     scrollGasPriceOracle: "0x247969F4fad93a33d4826046bc3eAE0D36BdE548",
-    modeCrossDomainMessenger: "0xc19a60d9E8C27B9A43527c3283B4dd8eDC8bE15C",
-    modeStandardBridge: "0xbC5C679879B2965296756CD959C3C739769995E2",
 
     // https://github.com/maticnetwork/static/blob/master/network/testnet/amoy/index.json
     polygonRootChainManager: "0x34F5A25B627f50Bb3f5cAb72807c4D4F405a9232",
     polygonFxRoot: "0x0E13EBEdDb8cf9f5987512d5E081FdC2F5b0991e",
     polygonERC20Predicate: "0x4258C75b752c812B7Fa586bdeb259f2d4bd17f4F",
     polygonRegistry: "0xfE92F7c3a701e43d8479738c8844bCc555b9e5CD",
     polygonDepositManager: "0x44Ad17990F9128C6d823Ee10dB7F0A5d40a731A4",
+  },
+};
 
-    // https://docs.lisk.com/contracts
-    liskCrossDomainMessenger: "0x857824E6234f7733ecA4e9A76804fd1afa1A3A2C",
-    liskStandardBridge: "0x1Fb30e446eA791cd1f011675E5F3f5311b70faF5",
-
-    // https://docs.blast.io/building/contracts
-    l1BlastBridge: "0xc644cc19d2A9388b71dd1dEde07cFFC73237Dca8",
-    blastCrossDomainMessenger: "0x9338F298F29D3918D5D1Feb209aeB9915CC96333",
-    blastStandardBridge: "0xDeDa8D3CCf044fE2A16217846B6e1f1cfD8e122f",
+export const OP_STACK_ADDRESS_MAP: {
```

Comment:
Note also we use a lot of these contract addresses in the relayer repository as well, and there are duplicate definitions for them. I think a subsequent step could be to relocate these defs over to the constants repo, so we can define them centrally and inherit in the contracts and relayer repositories. That eases new OP stack chain introductions and helps to reduce risk.